### DISCUSSION TITLE
kimai default nginx configuation breaks php for all other virutalhosts

### DISCUSSION THREAD

**Discussion by gdevenyi** [kimai/kimai#3214] [General]
**Describe the bug**
This line:
```
fastcgi_param PHP_ADMIN_VALUE "open_basedir=$document_root/..:/tmp/";
```

Breaks the PHP configuration for any other virtualhosts on nginx, since it limits PHP to only working in the kimai install directory

I suggest modifying the nginx configuration to be a virtualhost config rather than a default one, and test it against a server with existing installs.

**To Reproduce**
1. Install kimai in a virtualhost
2. Try to use PHP in another virtualhost

**Logfile**
```
Unknown: open_basedir restriction in effect. File(/opt/ldap-user-manager/www/index.php) is not within the allowed path(s): (/opt/kimai2/public/..:/tmp/)
```

**Additional context**
 - Kimai version: 1.18.2 stable 
 - PHP version: 8.0.16-1+ubuntu20.04.1+deb.sury.org+1
 - Device: Ubuntu 20.04 server
 - Browser Chrome 98.0.4758.80

**Comment by kevinpapst** [comment]
Not sure if you mean this page here: https://www.kimai.org/documentation/webserver-configuration.html which clearly says 
> This is only an example, that needs to be adapted to your needs (eg. path and PHP-FPM version).

but what about sending a PR to make it better?

Go here, click edit... https://github.com/kimai/www.kimai.org/blob/master/_documentation/webserver-configuration.md

**Reply by gdevenyi** [reply]
> This is only an example, that needs to be adapted to your needs (eg. path and PHP-FPM version).

Sure, but as someone doesn't "know" php, and expects to be able to mostly follow instructions as given, something that breaks other things isn't a great example.

**Reply by kevinpapst** [reply]
Usually I think manual hosting should only be handled by people understanding their business.
But then I guess everyone (including me) tends to turn off his brain partially when there is a step by step documentation.
So thanks for raising this issue and the PR!
